index.js

```diff
@@ -85,6 +238,12 @@ module.exports = function (app) {
           description: "When enabled the vessels position will be stored",
           default: false
         },
+        storeOtherVessels: {
```

Comment:
Does this mean "vessels" or "everything", like atons and sar aircraft (that appear in AIS data)?